src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java

```diff
@@ -81,28 +76,41 @@ public void create(EntityDescriptor descriptor,
 
     @Override
     protected void onBeforeSaveChecks(Object entity) {
-        getMultiLanguageString(entity).data().forEach((language, text) -> {
+        MultiLanguageString multiLanguageString = getMultiLanguageString(entity);
+        multiLanguageString.data().forEach((language, text) -> {
             if (Strings.areEqual(language, MultiLanguageString.FALLBACK_KEY)) {
                 return;
             }
-            if (supportedLanguages != null && !supportedLanguages.contains(language)) {
+            if (multiLanguageString.getValidLanguages() != null && !multiLanguageString.getValidLanguages().contains(language)) {
                 throw Exceptions.createHandled()
-                                .withNLSKey("MultiLanguageString.invalidLanguage")
-                                .set("language", language)
-                                .set("text", text)
-                                .set("field", getField().getName())
-                                .handle();
+                        .withNLSKey("MultiLanguageString.invalidLanguage")
```

Comment:
könntest du in sirius db mal n `mvn compile` ausführen und dann nochmal autoformat drüber laufen lassen? Durch das mvn compile zieht er sich die Codestyle Settings aus dem sirius-parent.